modules/parser/jvm/src/main/scala/io/circe/parser/ParserFactory.scala

```diff
@@ -0,0 +1,8 @@
+package io.circe.parser
+
+import io.circe.Parser
+import io.circe.jawn.JawnParser
+
+case class ParserFactory(maxValueSize: Option[Int], allowDuplicateKeys: Boolean) {
```

Comment:
Why is this needed? I'd prefer to keep the Scala.js and JVM APIs in circe-parser identical.